Docs new dashboard (#364)
Altinn/altinn-studio-docs: content/app/getting-started/create-app/_index.en.md

linktitle: Create app
description: How to create a new application in Altinn Studio.
weight: 150
---

Altinn Studio is used to create applications (apps).

> An app could be anything form simple forms to more complex applications with APIs, user interfaces, and everything in between.

Reviewer: ```suggestion
An app could be anything form a simple form to a more  complex application with APIs, user interfaces, and everything in between.
```

You create an app from the dashboard in Altinn Studio.

![Dashboard in Altinn Studio](overview.png "Dashboard - overview")

1. Click on "**ny app**"-button in the top right corner of the dashboard.

> 2. Select who the **eier** of the app should be. If you don't have access to any organisations you have to select your self and the app would be created in your "sandbox".

Reviewer: ```suggestion
2. Select the **eier** of the app. If you don't have access to any organisations, you have to select your self and the app would be created in your "sandbox".
```

> 3. Add the **navn** of the app. This name is used to identify the application, and also as the name of the repository, and in URL's and APIs.

Reviewer: ```suggestion
3. Add the **navn** of the app. This name is used to identify the application and will be used as the repository name, in URL's, and APIs.
```

   The name should be short and descriptive, for instance "sick-leave" or "local-election-2019".  
    _Name can **not** be changed after an app has been deployed._

> 4. Create the app by clicking "**Opprett**".

Reviewer: ```suggestion
4. Click "**Opprett**" to create the app.
```

![New app popup](new-app.png "Create new app")

> When the app is created you will be forwarded to the created application.

Reviewer: ```suggestion
Once the app is created, you will be forwarded to the created application.
```
